vrecord_policy.xml

```diff
@@ -0,0 +1,93 @@
+            <rule name="Is the video interlaced?" value="ScanType" tracktype="Video" operator="=">Interlaced</rule>
+            <rule name="Is the comression mode lossless?" value="Compression_Mode" tracktype="Video" operator="=">Lossless</rule>
+            <rule name="Is max slice count at least 4?" value="extra/MaxSlicesCount" tracktype="Video" operator="&gt;=">4</rule>
+            <rule name="Is error detection type per slice?" value="extra/ErrorDetectionType" tracktype="Video" operator="=">Per slice</rule>
+            <rule name="Is the video stream first?" value="StreamOrder" tracktype="Video" operator="=">0</rule>
+            <rule name="Is the colour description present?" value="colour_description_present" tracktype="Video" operator="=">Yes</rule>
+            <rule name="Are the transfer characteristics BT.709?" value="transfer_characteristics" tracktype="Video" operator="=">BT.709</rule>
+            <policy type="or" name="Is the bit depth 8 or 10?">
+                <rule name="Is the bit depth 8?" value="BitDepth" tracktype="Video" operator="=">8</rule>
+                <rule name="Is the bit depth 10?" value="BitDepth" tracktype="Video" operator="=">10</rule>
+            </policy>
+            <policy type="or" name="Is it NTSC SD or PAL SD?">
+                <policy type="and" name="Is it NTSC?">
+                    <rule name="NTSC height 486?" value="Height" tracktype="Video" operator="=">486</rule>
+                    <rule name="NTSC Width 720?" value="Width" tracktype="Video" operator="=">720</rule>
+                    <rule name="NTSC FrameRate 29.970?" value="FrameRate" tracktype="Video" operator="=">29.970</rule>
+                    <rule name="NTSC BitRate Mode CBR?" value="BitRate_Mode" tracktype="Video" ooperator="=">CBR</rule>
+                    <rule name="NTSC pixel aspect ratio 0.900?" value="PixelAspectRatio" tracktype="Video" operator="=">0.900</rule>
+                    <rule name="NTSC scan order bottom field first?" value="ScanOrder" tracktype="Video" operator="=">BFF</rule>
+                    <rule name="NTSC colour primaries BT.601 NTSC?" value="colour_primaries" tracktype="Video" operator="=">BT.601 NTSC</rule>
+                    <rule name="Are the matrix coefficients BT.601?" value="matrix_coefficients" tracktype="Video" operator="=">BT.601</rule>
+                </policy>
+                <policy type="and" name="Is it PAL?">
+                    <rule name="PAL height 576?" value="Height" tracktype="Video" operator="=">576</rule>
+                    <rule name="PAL Width 720?" value="Width" tracktype="Video" operator="=">720</rule>
+                    <rule name="PAL FrameRate 25.000?" value="FrameRate" tracktype="Video" operator="=">25.000</rule>
+                    <rule name="PAL BitRate Mode VBR?" value="BitRate_Mode" tracktype="Video" operator="=">VBR</rule>
```

Comment:
As with line 67 - is this intentional, should both be looking for VBR?

Reply:
I guess the uncompressed should be CBR, but for the others maybe the test is not needed. For instance if someone made an ffv1 recording of black, the result may intentionally be CBR and not VBR since all the frames would be identical.

Reply:
@dericed, to confirm, you mean to remove lines 67 and 77 for this ffv1 policy, but to include CBR when the policy gets expanded to handle uncompressed files made in vrecord?

Reply:
yep